.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+     - master
+
+env:
+  DISPLAY: :100.0
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12, 14, 16]
```

Comment:
According to https://nodejs.org/en/about/releases/, v12 hits end-of-life around now, and v18 is out. How about using versions `[14, 16, 18]`?

Reply:
@jeffcarp 
Thanks for your comment! I removed version 12.
https://github.com/jsdom/abab/pull/77/commits/228a57ef1b388baf5aa1706f7e17d4e65f2f307b

Regarding version 18, I tried it but unit tests stop.
https://github.com/is2ei/abab/runs/6205318259?check_suite_focus=true
I'm not sure the reason but libraries might be too old. I can investigate it but it would be another issue. I'd like to focus on this PR to adding GitHub Actions.

Reply:
Testing v14 and v16 for now sgtm. Can you file a bug for testing v18 in CI?